ops/ansible/playbooks-ccs/files/prod-bluebutton-appserver-public-cert.pem

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
+MIIFpTCCA42gAwIBAgIEH/WYQjANBgkqhkiG9w0BAQsFADAWMRQwEgYDVQQDEwti
+ZmQuY21zLmdvdjAeFw0xOTA5MTgwMjAyMzRaFw0yMTA5MTcwMjAyMzRaMBYxFDAS
+BgNVBAMTC2JmZC5jbXMuZ292MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKC
+AgEAq9Okj4oJcYCKNvjbjAt7l5KhqmKAJAbwZ+jMB7R6DfLUSe2d9StnZw1FpIks
+LyyvJsOzU3zIQPl4IJWJir7V1Buaok4wFeEs0S/B6eQRbF4/kpZPZPyv9vqsXqxd
+2BNh0QovZV9d+WvKUClv/baNA2vu04PSwcqi+JnSDmCokE/PSkSwrRHVF+7hTFLc
+xvJvKitFy2yN/8lvNnq0SiOed7KtOCrSoyNFRO/eQ91orxpVJBAq0Br2PKSJcl3Q
+Eavs1As1nJJs9tib1u/mBMrJK04BLm3pTGN0n/QmRWLQ0b5vB8rcT7FiM4Q1D1o8
+zqCMBg7Di88s01LSdagbuFP7sgH7DsBs4XufWZoh3PHgLk36FqOZM3t4rjtaeHzO
+bnmigbL8TiQxmsHa82fOjdtCzM6LPU59mXBUzxv///zChK9Q484u0dT0DfUp0STt
+TX5snloTdCt9P3ZES4MU7Q6f/4IhAWbC1BfdxohzW0euuyKkmMCxaK5m8pFX5z2P
+2WvK7V6GW8Bi3ALsu7yF+ffuUVBjTaZJP4ENrN6ZeqQfzO51y+8W6S2oYeFALph8
+/r0E0Hf4qLzbflnQMU0BkxX6rGyKhGAtPFUkyDmUWOFsqhTPMWyz2DvtHxQpvUow
+CtlKOArb3MklZT6wMKoDL3agq22K0ABtiRqhacF6P5QvP5UCAwEAAaOB+jCB9zCB
+1QYDVR0RBIHNMIHKghFwcm9kLmJmZGNsb3VkLm5ldIIVbWN0LnByb2QuYmZkY2xv
+dWQubmV0ghVkcGMucHJvZC5iZmRjbG91ZC5uZXSCFmJjZGEucHJvZC5iZmRjbG91
+ZC5uZXSCFGJiLnByb2QuYmZkY2xvdWQubmV0gjppbnRlcm5hbC1wZGN3MTBsYjAx
+LTE5NTEyMTIyNjIudXMtZWFzdC0xLmVsYi5hbWF6b25hd3MuY29tggtiZmQuY21z
+LmdvdoIQcHJvZC5iZmQuY21zLmdvdjAdBgNVHQ4EFgQUl/ZpB/h0G1jmKtV2KIn+
+OduHH+YwDQYJKoZIhvcNAQELBQADggIBACPyWX0vDT6O04AYLJBzDAxef30K/YRt
+BKCi12ppZoHG6KsTUGtid6drA54sZfXSUJwfZR8XhqLcOnQV0sKNqAzHaUAEwmF7
+OBJMHbXorcM181atp3B/tUFQ2vIY/dms7U9EWF7Ia+F7pdyQXOFvwJglsxalO9in
+fonr3cpH811QI/OFHpKzXVYuTYC6WS+QZHHTFHy2xppM+OsvOcoQja0QESgRYG8x
+muQwuNci9DFMWELbvZ80W+wf3E2u0l0Jms9TAfreQaGBChuUilQDn3Nx1zfUypQB
+/WGp3s5hIT0bZqbd1e4rwES42Ve0T87lYQ1A2+kc5BN7kaSC5q4Ijhuhr9bdK9N3
+4rHa9cNwgf1bFDp7me+3dU9Nw3ujKaB7xOiIVKwty3euVorNvQTbN+adqW4c7Dv+
+abFVaQT+5hkgFS9FILVn/FloBW21mrxzvkB0wwkdHEgfc1kcdlpzq68VBHYyfgUI
+WmFEIgqMBm1NzsNiEtyOEApWMNHSOylUAkaWiBVQSy5NXYAZZD7187LmToaL8Pi9
+MEU2yxihsliXuXtnBeYJS+tM0q8sKZFPytVc3VGu7vFtQr04OSlIdcLfuNmNJ/QQ
+68Hweb4SSYPGLD/zSYCJm7MNOC+/iAuNkVqRuZs3kSNdaISQfSRyZMmWAabpvcm7
+4YpGcZVKl9m8
+-----END CERTIFICATE-----
```

Comment:
Reviewing the cert:

```
$ openssl x509 -text -in ops/ansible/playbooks-ccs/files/prod-bluebutton-appserver-public-cert.pem | less
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 536188994 (0x1ff59842)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=bfd.cms.gov
        Validity
            Not Before: Sep 18 02:02:34 2019 GMT
            Not After : Sep 17 02:02:34 2021 GMT
        Subject: CN=bfd.cms.gov
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:ab:d3:a4:8f:8a:09:71:80:8a:36:f8:db:8c:0b:
...
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Alternative Name:
                DNS:prod.bfdcloud.net, DNS:mct.prod.bfdcloud.net, DNS:dpc.prod.bfdcloud.net, DNS:bcda.prod.bfdcloud.net, DNS:bb.prod.bfdcloud.net, DNS:internal-pdcw10lb01-1951212262.us-east-1.elb.amazonaws.com, DNS:bfd.cms.gov, DNS:prod.bfd.cms.gov
            X509v3 Subject Key Identifier:
                97:F6:69:07:F8:74:1B:58:E6:2A:D5:76:28:89:FE:39:DB:87:1F:E6
    Signature Algorithm: sha256WithRSAEncryption
         23:f2:59:7d:2f:0d:3e:8e:d3:80:18:2c:90:73:0c:0c:5e:7f:
...
```

Looks good to me.

For comparison, here's the old prod cert:

```
$ openssl x509 -text -in ops/ansible/playbooks-healthapt/files/server_data_server_env_prod_certificate.pem | less
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 1139558869 (0x43ec49d5)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=internal-pdcw10lb01-1951212262.us-east-1.elb.amazonaws.com
        Validity
            Not Before: Mar 21 17:37:22 2018 GMT
            Not After : Mar 20 17:37:22 2021 GMT
        Subject: CN=internal-pdcw10lb01-1951212262.us-east-1.elb.amazonaws.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:83:aa:b4:63:b8:e0:ef:eb:1f:55:a2:04:90:88:
...
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Key Identifier:
                3B:E1:00:A1:AF:47:D6:32:4B:C3:6C:EE:DE:02:20:39:56:D2:E4:51
    Signature Algorithm: sha256WithRSAEncryption
         3e:30:4f:9b:b2:d6:3a:80:5e:b0:05:4b:69:40:31:b5:3d:f0:
...
```

Reply:
Mucho gracias for the check.